platform/broadcom/sonic-platform-modules-dell/common/dell_i2c_utils.sh

```diff
@@ -60,3 +60,60 @@ i2c_poll_bus_exists() {
     fi
 }
 
+# Perform an i2c mux device create
+# Input is of the form:
+# "echo [mux driver] <i2c-mux-address> >  <i2c-bus/operation>" i2c-channel-first
+# where operation = "new_device"
+# i2c-channel-first is the first of the 8 channels that this mux should create
+i2c_mux_create() {
```

Comment:
for me, this piece of code is a little bit hard to understand. since the i2c tree is fixed on each platform, it would be better to have some more predefined numbers instead of number manipulation here.

the function could be simpler like i2c_mux_create \<root\> \<sub_root\> \<first\>
so that in the script you only need to call the function e.g. i2c_mux_create 4 0x71 10 or i2c_mux_create 7 0x71 50 or i2c_mux_create 7 0x72 58

the echo command and the details of the retry could be inside the function

Reply:
Addressed this.